src/components/popup/TokenLogo.tsx

```diff
@@ -7,13 +7,17 @@ import { AR_LOGO, AR_PROCESS_ID, type TokenInfo } from "~tokens/aoTokens/ao";
 import { FULL_HISTORY, useGateway } from "~gateways/wayfinder";
 import { concatGatewayURL } from "~gateways/utils";
 
+import tokenPlaceholder from "url:/assets/images/tokens/loading-token.svg?no-inline";
```

Comment:
Custom placeholder for tokens. Updated the color schema to match the `Skeleton` component used in the tokens list.